specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-01-01/customer-insights.json

```diff
@@ -330,29 +404,26 @@
         ],
         "responses": {
           "200": {
-            "description": "OK. Successfully get all the hubs in the subscription.",
+            "description": "OK. Successfully get the connector operation.",
             "schema": {
-              "$ref": "#/definitions/HubListResult"
+              "$ref": "#/definitions/ConnectorResourceFormat"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/profiles/{profileName}": {
-      "put": {
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/segments/": {
```

Comment:
Feedback about this API -
1. remove the / at the end of the API as this is a collection GET
2. provisioningState property needs to support Succeeded, Failed and Canceled terminal states.
3. What is the name proerpty inside properties. name is already there as a top level property outside properties.
4. Mark the read only properties appropriately. Ex - segmentId, created, version is most likely readonly.
5. rename created to creationTime and make the format as date-time to make is ISO8601 compliant.